tests/components/twitch/fixtures/check_user_subscription.json

```diff
@@ -1,3 +1,4 @@
 {
-  "is_gift": true
+  "is_gift": true,
+  "tier": 2000
```

Comment:
Is this a real value? You probably use Twitch more than me, but I know that they have tier 1, 2 and 3 right?

Reply:
Yes, the tier values are 1000, 2000, and 3000.  I have no idea why.  See https://dev.twitch.tv/docs/api/reference/#check-user-subscription